homestar-runtime/src/network/swarm.rs

```diff
@@ -131,6 +131,9 @@ pub(crate) fn init(
     // Listen-on given address
     swarm.listen_on(settings.listen_address.to_string().parse()?)?;
 
+    // Set Kademlia server mode
+    swarm.behaviour_mut().kademlia.set_mode(Some(Mode::Server));
```

Comment:
minor: maybe bring this in as kad::Mode (via self), b/c on it's own, it's a weirdly named thing.

Reply:
Yep, good call! Added that. 👍